store/sqlstore/channel_store.go

```diff
@@ -1046,6 +1046,35 @@ func (s SqlChannelStore) GetMoreChannels(teamId string, userId string, offset in
 	return channels, nil
 }
 
+func (s SqlChannelStore) GetPrivateChannelsForTeam(teamId string, offset int, limit int) (*model.ChannelList, *model.AppError) {
+	channels := &model.ChannelList{}
+
+	_, err := s.GetReplica().Select(channels, `
```

Comment:
small nit 1/5 - maybe rewrite this using squirrel? We are trying to keep all non trivial queries written with it

Reply:
done, switched to squirrel.